test/dropRepeatsBy.js

```diff
@@ -0,0 +1,35 @@
+var R = require('../source/index.js');
+var eq = require('./shared/eq.js');
+
+
+describe('dropRepeatsBy', function() {
+  var objs = [
+    {i: 1}, {i: 2}, {i: 3}, {i: -4}, {i: 5}, {i: 3}
+  ];
+  var objs2 = [
+    {i: 1}, {i: -1}, {i: 1}, {i: 2}, {i: 3},
+    {i: 3}, {i: -4}, {i: 4}, {i: 5}, {i: 3}
+  ];
+  var pred = R.compose(R.evolve({ i: Math.abs }), R.dissoc('n'));
```

Comment:
I would rather write this out explicitly.  And I definitely wouldn't name it `pred`, as this is an object-transformation function, not a predicate.

Perhaps something like

```js
  var convert = ({i, n, ...rest}) => ({i : Math.abs(i), ...rest});
```

Reply:
You're right, `pred` is not accurate naming. Changes done as requested.